uber/config.py

```diff
@@ -136,6 +136,19 @@ def SUPPORTER_COUNT(self):
                                                 sa.Attendee.amount_paid >= self.SUPPORTER_LEVEL).count()
             return individual_supporters + group_supporters
 
+    @property
+    def SQLALCHEMY_FINAL_URL(self):
```

Comment:
You should use docstrings for comments like this.  So instead of prefixing each line with a ``#`` you should usea triple-quoted string like

```
"""
support reading the DB connection....
"""
```